.travis.yml

```diff
@@ -3,9 +3,8 @@ os:
 language: c
 
 before_install:
-  - sudo apt-get install qemu-kvm qemu qemu-user-static
-  - curl https://api.github.com/repos/vitasdk/autobuilds/releases  | grep browser_download_url | grep linux | head -n 1 | cut -d '"' -f 4 | xargs wget
-  - tar xjf *.tar.bz2
+  - sudo apt-get install qemu-kvm qemu qemu-user-static jq
+  - curl -L "https://api.github.com/repos/vitasdk/autobuilds/releases?per_page=100" | jq -r '[.[] | select(.prerelease | not)][].assets[].browser_download_url' | grep 'master' | grep 'x86_64-linux' | head -n 1 | xargs curl -L | tar xj
   - export VITASDK=$PWD/vitasdk
   - export PATH=$VITASDK/bin:$PATH
   - wget https://releases.linaro.org/archive/15.02/components/toolchain/binaries/arm-linux-gnueabihf/gcc-linaro-4.9-2015.02-3-x86_64_arm-linux-gnueabihf.tar.xz
```

Comment:
While I'm here, any reason why it uses this specific version of GCC? I've been building it in a Debian 10-based docker image and it works just fine with GCC 8.